08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep

```diff
@@ -0,0 +1,37 @@
+@description('Specify the name of automation account to be created.\r\nUse only alphanumerics and hyphens.\r\nThe name must not start or end with alphanumeric and must be 6 - 50 characters in length.\r\n')
+param automationAccountName string
+
+@description('Enter location. If you leave this field blank resource group location would be used.')
+param location string = resourceGroup().location
+
+@description('Specify the name of automation account\'s variable')
+param automationAccountVariableName string
+
+@description('Specify the value of automation account\'s variable')
+param automationAccountVariableValue string
+
+@description('Specify the description of automation account\'s variable')
+param automationAccountVariableDescription string
+
+resource automationAccount 'Microsoft.Automation/automationAccounts@2020-01-13-preview' = {
```

Comment:
Any particular reason for using this preview API? I see updated and stable version available:  api-version=2021-06-22

Reply:
No there is no particular reason,

I have updated the version to 22-08-08